kubernetes/test-infra/eks/node-config/aws_controller_deps.yaml

```diff
@@ -0,0 +1,572 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.4.0
+  creationTimestamp: null
+  labels:
+    app.kubernetes.io/name: aws-load-balancer-controller
+  name: targetgroupbindings.elbv2.k8s.aws
```

Comment:
This file is from the AWS LoadBalancer Controller docs: https://raw.githubusercontent.com/kubernetes-sigs/aws-load-balancer-controller/v2.1.0/docs/install/v2_1_0_full.yaml

I removed the service account and deployment, and made them managed by the kubernetes provider. Eventually, I'd like to manage some of the other pieces too. That is, if we decide to keep this controller around.